### DISCUSSION TITLE
user creation via stalwart-cli

### DISCUSSION THREAD

**Discussion by zvasilev** [stalwartlabs/jmap-server#53] [General]
Hello [@mdecimus](https://twitter.com/mdecimus),

I used the initially released jmap server and utilized this command to create a user:
stalwart-cli -u https://mailtemi.eu -c "admin_password" account create test@mailtemi.eu q "test"

However, after installing the complete mail server, it appears that these commands no longer exist. I've only come across the "import" command in stalwart-cli, which seems to be the most relevant now. Could you please direct me to documentation on how to create a user via stalwart-cli?

Regards,
Zhivko

**Comment by mdecimus** [comment]
Hi @zvasilev ,

The new version delegates user management to an external directory, this means that account creation has to be done on your directory server which could be an SQL or LDAP database. Please check the [FAQ](https://stalw.art/docs/faq#how-do-i-create-and-manage-users) for more details.
I suggest to install Stalwart and when asked what's your directory select "I don't have one" so the installer creates an SQLite auth database for you.
